source/css/pattern-scaffolding.css

```diff
@@ -99,6 +99,15 @@
 	font-size: 1rem;
 }
 
+/**
+* used for input 
+**/
+.box-with-border {
+	border: 1px solid rgba(0, 0, 0, 0.06);
+	padding: 0 1rem 1rem;
+	margin: 0 0 2rem;
+}
+
```

Comment:
This is a file provided by pattern lab – please move those declarations to `source/css/pattern-scaffolding-project-specific.scss`

Reply:
oh, thx :)